book/src/slasher.md

```diff
@@ -8,10 +8,10 @@ extra income for your validators. However it is currently only recommended for e
 of the immaturity of the slasher UX and the extra resources required.
 
 ## Minimum System Requirements
-
+[comment]: <> (change to 512GB as below stated 300K validators, but mainnet is now 600K. Please correct me if this is wrong)
```

Comment:
```suggestion
```

I ran this past @michaelsproul and he's indicated that 256GB is still adequate. In fact 175GB would work for 600k validators. Perhaps we just leave it as-is for now.